openwillis/measures/text/util/characteristics_util.py

```diff
@@ -176,23 +236,209 @@ def get_mattr(word):
 
     return mattr
 
-def get_stats(summ_df, ros, file_dur, pause_list, measures):
+
+def get_sentiment(df_list, text_list, measures):
+    """
+    ------------------------------------------------------------------------------------------------------
+
+    This function calculates the sentiment scores of the input text using
+     VADER, and adds them to the output dataframe summ_df.
+
+    Parameters:
+    ...........
+    df_list: list
+        List of pandas dataframes.
+            word_df, phrase_df, turn_df, summ_df
+    text_list: list
+        List of transcribed text.
+            split into words, phrases, turns, and full text.
+    measures: dict
+        A dictionary containing the names of the columns in the output dataframes.
+
+    Returns:
+    ...........
+    df_list: list
+        List of updated pandas dataframes.
+
+    ------------------------------------------------------------------------------------------------------
+    """
+    word_df, phrase_df, turn_df, summ_df = df_list
+    word_list, phrase_list, turn_list, full_text = text_list
+
+    sentiment = SentimentIntensityAnalyzer()
+
+    # column names
+    cols = [
+        measures["neg"],
+        measures["neu"],
+        measures["pos"],
+        measures["compound"],
+        measures["speech_mattr"],
+    ]
+
+    # word-level analysis
```

Comment:
do we need to handle exception here?

Reply:
[try-except block in sentiment analysis](https://github.com/bklynhlth/openwillis/pull/22/commits/ab90b9106ba9c0ba3bd91fc0d1e6bb9e81050a60)